Makefile

```diff
@@ -43,11 +43,11 @@ test::
 	@# Help: Recursively $@ in all subdirs 
 	@$(MAKE) RULE=$@ .recurse
 
-publish::  
+publish::
 	@# Help: Recursively $@ in all subdirs 
 	@$(MAKE) RULE=$@ .recurse
 
-set-versions::  
+set-versions:  
```

Comment:
@daw3rd , why do we use `::` in other rules, we don't need multiple rules for the same target, each executed independently?
and why do you remove it here? what is the difference?

Reply:
The :: allows sub-makefiles to override to extend the function.  I'm not really sure we need this anymore.  However, for set-versions, I was just trying to make it consistent across all makefiles.  I guess when these are defined in the leaf makefiles, they can be whatever the leaf wants to use.  Problems seemed to arise when being inconsistent with : vs :: in makefiles that tried to override/extend one of the .make.* targets.  I'm beginning to think, the .make.* targets should all go back to : so they are thought of more as sub-routines that extendible methods.